src/main/java/pl/Lukasz/Calories/Entity/Meals.java

```diff
@@ -0,0 +1,78 @@
+package pl.Lukasz.Calories.Entity;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name="meals")
+public class Meals {
+
+    private Integer id;
+    private String date;
```

Comment:
Please use better data types for date and time (you can have just one column for that - DateTime)